gui/browsable/src/RProvider.cxx

```diff
@@ -509,10 +509,6 @@ class RDefaultProvider : public RProvider {
    RDefaultProvider()
    {
       // TODO: let read from rootrc or any other files
-      RegisterClass("ROOT::Experimental::RH1D", "sap-icon://bar-chart", "", "", "libROOTHistDrawProvider");
-      RegisterClass("ROOT::Experimental::RH2D", "sap-icon://pixelate", "", "", "libROOTHistDrawProvider");
-      RegisterClass("ROOT::Experimental::RH3D", "sap-icon://product", "", "", "libROOTHistDrawProvider");
-      RegisterClass("ROOT::Experimental::RCanvas", "sap-icon://business-objects-experience", "", "", "libROOTHistDrawProvider");
```

Comment:
That smell odd to remove a line about `RCanvas` (but it also smell odd-ish that it ask for `libROOTHistDrawProvider`)

Reply:
Yes, this is automatic loading of libs to handle `RH1D`, `RH2D`, `RH3D` classes. 
This should be removed from RBrowser as well, but this can be done after merging this PR